### PR TITLE
Restructure code with configuration token support

### PR DESCRIPTION
**Pull request by TarasMaliuchenko**
Implement configuration token
Fix unit tests

### CODE REVIEW

**Comment by AnastasiiaRozlyvan** [comment]
It would be nice to remove .github/workflows and [snapshot_updater](https://github.com/CiscoSecurity/tr-05-serverless-apivoid/blob/CCTRI-1941/code/snapshot_updater.py)

File: README.md [hunk 1]
```diff
@@ -24,7 +27,7 @@ You can perform two kinds of testing:
 
 - Run static code analysis checking for any semantic discrepancies and [PEP 8] https://www.python.org/dev/peps/pep-0008/) compliance:
```
Comment (AnastasiiaRozlyvan) [inline README.md:28]: You can fix this link by adding a parenthesis before **https:**...

**Comment by AlinaBlahun** [comment]
> It would be nice to remove .github/workflows and [snapshot_updater](https://github.com/CiscoSecurity/tr-05-serverless-apivoid/blob/CCTRI-1941/code/snapshot_updater.py)

Unfortunately, the same in https://github.com/CiscoSecurity/tr-05-serverless-cyberprotect/tree/develop/code

File: README.md (hunk 1 above)
Comment (TarasMaliuchenko) [reply README.md]: Fixed

Comment (AnastasiiaRozlyvan) [reply README.md]: It seems to be still broken because of extra space, in markdown syntax it should be like this `[](url)`

Comment (TarasMaliuchenko) [reply README.md]: Thank you. Fixed

**Review by AnastasiiaRozlyvan — APPROVED** [review]

**Review by AlinaBlahun — APPROVED** [review]

**Pull request merged**